coapthon/client/helperclient.py

```diff
@@ -120,7 +120,7 @@ def get_non(self, path, callback=None, timeout=None, **kwargs):  # pragma: no co
         :return: the response
         """
         request = self.mk_request_non(defines.Codes.GET, path)
-        #request.token = generate_random_token(2)
+        request.token = generate_random_token(4)
```

Comment:
IIRC the reason for this change was to make the message sent by the autoctt client identical to the one from DeviceSpy. This might not be strictly necessary.

@FrankZRS , would you mind reverting this change and testing to see if the script overall still works?

Reply:
Yeah you're right, it still works with the change reverted.

Reply:
In that case I think we should preserve the original code, as there is no reason for the change.